crates/revm/src/handler/mainnet/post_execution.rs

```diff
@@ -49,11 +66,7 @@ pub fn reward_beneficiary<EvmWiringT: EvmWiring, SPEC: Spec>(
         .map_err(EVMError::Database)?;
 
     coinbase_account.data.mark_touch();
```

Comment:
We are skipping touch on coinbase account.

Geth code looks like a edge case:
https://github.com/ethereum/go-ethereum/blob/ae707445f54972f4fddc7c6cf8ba2de083636e50/core/state/state_object.go#L452-L459

This can potentially be a bug for EIP-161 State clear. The trigger is if coinbase is empty_account (Created before EIP-161) and there is no fee for this tx.